src/components/LoginForm/LoginForm.jsx

```diff
@@ -25,13 +25,40 @@ import passwordSVG from '../../assets/icons/lock.svg';
 import { Formik, Form, Field } from 'formik';
 import * as Yup from 'yup';
 import { Link } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { setError, setIsAuth, setUserName, setUserToken } from '../../redux/session/sessionSlice';
+import axios from 'axios';
 
 const LoginSchema = Yup.object().shape({
   email: Yup.string().email('Invalid email').required(''),
   password: Yup.string().required(''),
 });
 
 const LoginForm = () => {
+  const dispatch = useDispatch();
+  const handleSubmit = async (values, { setSubmitting }) => {
+    const dataToSend = values;
+    try {
+      const response = await axios.post(
+        'https://pocketbook-basket-clam.cyclic.app/api/users/login',
+        JSON.stringify(dataToSend),
+        {
+          headers: {
+            'Content-Type': 'application/json',
+          },
+        }
+      );
+      dispatch(setUserName(response.data.user.name));
+      dispatch(setUserToken(response.data.user.token));
+    } catch (error) {
+      dispatch(setError(error.response.data.message));
+      console.log('Error:', error.response.data.message);
+    } finally {
+      dispatch(setIsAuth(true));
```

Comment:
Nie możesz tego dodać do finally - bo wykona się za każdym razem, nawet jeśli użytkownik poda złe hasło to ustawi setIsAuth na true, najlepiej ustaw tego dispatcha razem z innymi dispatchami w warunku czyli jesli response.code jest odpowiedni